.github/workflows/test.yaml

```
@@ -13,6 +13,8 @@ jobs:
   setup:
     name: Set up
     runs-on: ubuntu-latest
+    if: |
```

actionlint (1.7.4)

if: condition "${{ !startsWith(github.event.pull_request.title, 'build:') && !startsWith(github.event.pull_request.title, 'bump') }}\n" is always evaluated to true because extra characters are around ${{ }}

(if-cond)